src/js/components/Pagination/Pagination.js

```diff
@@ -148,10 +161,7 @@ const Pagination = forwardRef(
         'aria-current': page === activePage ? 'page' : undefined,
         page,
         onClick: event => {
-          event.persist();
-          const adjustedEvent = event;
-          adjustedEvent.page = page;
-          handleClick(adjustedEvent);
+          handleClick(event, page);
```

Comment:
```suggestion
const nextPage = page;
          handleClick(event, nextPage);
```
maybe this for consistency with next/previous? or is it redundant to reassign

Reply:
In this case I think the code reads just fine and doesn't need the re-assignment. Especially, if you adopt my next suggestion.